### ISSUE TITLE
documentation 

### ISSUE THREAD

**Issue by potatochick2020** [potatochick2020/Sudoku-core#7]
Provide Documentation on each function on what it does. Please also provide documentation on elaboration on the algorithm especially for 'SudokuSolution()' and 'initialize()'

**Comment by kavyabhat02** [comment]
Hi, can I take this up?

**Comment by potatochick2020** [comment]
Yes sure, but please make sure you could understand *SudokuSolution()* and *initialize()* these two function before you start writing. 

Wish you have fun in hacktoberfest and give the project a star if you like it~

**Comment by kavyabhat02** [comment]
Really sorry, but I don't think I'll be able to work on this for a few more days because of other work that has come up.
I'll be unassigning myself, feel free to assign this issue to someone else!

**Comment by Prasad-D-Ware** [comment]
can i work on this ?

**Comment by potatochick2020** [comment]
Yes sure. It might be good if you can work after [This issue has done in weekend](https://github.com/potatochick2020/Sudoku-core/issues/2) and look into the SudokuSolution() and initialize() these two function at this moment?

**Comment by Prasad-D-Ware** [comment]
Okay ! Sure

**Comment by Divyanshu975** [comment]
Can I also take this up ?
